### PR TITLE
B-21525: Update email: ppm has been processed

### PR DESCRIPTION
**Pull request by WeatherfordAaron**
## Testing Steps I took
1. Start the local server with the following command:
```
EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run
```
2. Start the local client.
3. As a Service Member, create a new PPM move and send to Service Counsellor. Make sure you try this at least once with one of each of the following categories:
- Marines, Navy, or Coast Guard
- Army
- Air Force or Space Force

_Note: Make sure you use an email address that you have access to and have a checkmark in the "Email" checkbox for preferred contact method._

4. As Service Counsellor, update the orders and approve the move and be sure to try at least one time with Actual Expense Reimbursement checked and at least one time with it unchecked.
5. As the Service Member, make sure to finish the process to that it shows up in the PPM Closeout Queue for the Service Counsellor role.
6. As a Closeout Counsellor, make sure you can see the move under the PPM Closeout tab and open it. You may have to click on the "Review Documents" button and approve any of the items that need approval.
7. Open your email and you should see several emails.. one of which will have the subject:
```
Your Personally Procured Move (PPM) closeout has been processed and is now available for your review.
```

You should see email body that follows the appropriate conditions in the Agility Ticket [B-21525](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21525)

___
# Screenshots of the email bodies
## Navy, Marine Corps, Coast Guard (AER)
Screenshot 2024-11-01 at 11 02 09 AM

## Nave, Marine Corps, Coast Guard (not AER)
Screenshot 2024-11-01 at 10 59 49 AM

## Army (AER)
Screenshot 2024-11-01 at 11 09 29 AM

## Army (not AER)
Screenshot 2024-11-01 at 11 07 31 AM

## Air Force, Space Force (AER)
Screenshot 2024-11-01 at 11 15 27 AM

## Air Force, Space Force (not AER)
Screenshot 2024-11-01 at 11 16 43 AM

### CODE REVIEW

File: src/pages/MyMove/PPM/Closeout/Feedback/Feedback.test.jsx
Comment (KonstanceH) [inline src/pages/MyMove/PPM/Closeout/Feedback/Feedback.test.jsx:1]: There  are some differences in this file compared to integration.
the spelling of displays is fixed in your int pr. Lines 82-88 aren't removed in your int PR.
And there are duplicate asserts on lines 105-114

File: pkg/handlers/ghcapi/move_task_order_test.go
[the remark this answers is not in the document]
Comment (WeatherfordAaron) [reply pkg/handlers/ghcapi/move_task_order_test.go:1]: It was pulled from main. I'm tracking that other PR that it's in and they are removing it (afaik)

File: src/pages/MyMove/PPM/Closeout/Feedback/Feedback.test.jsx
Comment (WeatherfordAaron) [reply src/pages/MyMove/PPM/Closeout/Feedback/Feedback.test.jsx:1]: I saw the dup assertions/tests and removed them but I cannot find where the word "displays" was spell corrected. Could you give me a line # on that one?

Comment (KonstanceH) [reply src/pages/MyMove/PPM/Closeout/Feedback/Feedback.test.jsx:1]: yea its on line 82. Its deleted on this branch but on the main branch the lines of of 82-88 weren't deleted and the first line has the spelling correction.
Screenshot 2024-11-06 at 8 26 21 AM

File: pkg/handlers/ghcapi/move_task_order_test.go
[the remark this answers is not in the document]
Comment (taeJungCaci) [reply pkg/handlers/ghcapi/move_task_order_test.go:1]: Just wondering how will it show up in this PR is it was pulled from Main?

File: pkg/notifications/constants.go [hunk 1]
```diff
@@ -1,7 +1,22 @@
 package notifications
 
+import "github.com/transcom/mymove/pkg/models"
+
 const OneSourceTransportationOfficeLink = "https://installations.militaryonesource.mil/search?program-service=2/view-by=ALL"
 const MyMoveLink = "https://my.move.mil/"
 const WashingtonHQServicesLink = "https://www.esd.whs.mil"
-
+const SmartVoucherLink = "https://smartvoucher.dfas.mil/"
```
Comment (taeJungCaci) [inline pkg/notifications/constants.go:8]: An empty line was deleted here - not sure if anyone actually cares, but just noting.

File: pkg/services/move_task_order/move_task_order_updater_test.go [hunk 2]
```diff
@@ -631,12 +631,14 @@ func (suite *MoveTaskOrderServiceSuite) TestMoveTaskOrderUpdater_MakeAvailableTo
...
+		startDate := time.Date(2020, time.January, 1, 12, 0, 0, 0, time.UTC)
+		endDate := time.Date(2020, time.December, 31, 12, 0, 0, 0, time.UTC)
 		contractYear := testdatagen.FetchOrMakeReContractYear(suite.DB(), testdatagen.Assertions{
 			ReContractYear: models.ReContractYear{
 				Contract:             contract,
 				ContractID:           contract.ID,
-				StartDate:            time.Now(),
-				EndDate:              time.Now().Add(time.Hour * 12),
+				StartDate:            startDate,
+				EndDate:              endDate,
```
Comment (taeJungCaci) [inline pkg/services/move_task_order/move_task_order_updater_test.go:641]: Not sure about these either - Not in the INT PR

File: pkg/notifications/constants.go (hunk 1 above)
Comment (WeatherfordAaron) [reply pkg/notifications/constants.go:8]: I'm not sure why this isn't registering it as the line-break moving down
but I'll add one there just to make this work

File: pkg/services/move_task_order/move_task_order_updater_test.go (hunk 2 above)
Comment (WeatherfordAaron) [reply pkg/services/move_task_order/move_task_order_updater_test.go]: corrected

File: pkg/handlers/ghcapi/move_task_order_test.go
[the remark this answers is not in the document]
Comment (WeatherfordAaron) [reply pkg/handlers/ghcapi/move_task_order_test.go:1]: corrected

File: src/pages/MyMove/PPM/Closeout/Feedback/Feedback.test.jsx
Comment (WeatherfordAaron) [reply src/pages/MyMove/PPM/Closeout/Feedback/Feedback.test.jsx:1]: corrected

File: pkg/handlers/ghcapi/move_task_order_test.go
Comment (KonstanceH) [inline pkg/handlers/ghcapi/move_task_order_test.go:1]: These changes should be reverted to match the current main

**Review by KonstanceH — APPROVED** [review]
LGTM

File: pkg/handlers/ghcapi/move_task_order_test.go
Comment (WeatherfordAaron) [reply pkg/handlers/ghcapi/move_task_order_test.go:1]: woops.. corrected. I was looking at the wrong file with the same name

**Pull request merged**